runtime/list_test.go

```diff
@@ -392,8 +392,8 @@ func TestListSetItem(t *testing.T) {
 }
 
 func TestListSort(t *testing.T) {
-	fun := newBuiltinFunction("TestListSetItem", func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
-		if raised := checkFunctionArgs(f, "TestListSetItem", args, ListType); raised != nil {
+	fun := newBuiltinFunction("TestListSort", func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
```

Comment:
To get some coverage over the listSort() code, you could make this test call list.sort() rather than *List.Sort(). Then you could test the arg validating logic as well.